plugins/heartbeats/scripts/heartbeats.py

```
@@ -0,0 +1,403 @@
+        "--crontab-file",
+        help="read/write this file instead of the real crontab (test injection point)",
+    )
+    parser.add_argument(
+        "--approve",
+        action="store_true",
+        help="required by `install`; without it install is a dry run",
+    )
+    return parser
+
+
+def _unit_header_names(lines):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_unit_header_names`

(ANN202)